src/components/NavBar/NavBarPresentationMobile.js

```diff
@@ -24,7 +24,7 @@ function NavBarPresentationMobile({ site }) {
   }
 
   const menuData = site?.menu || {}
-  const member = isMember({ user, sitename: site?.sitename })
+  const member = isMember({ user, sitename: site.sitename })
```

Comment:
is it safe to remove the optional chaining operator here ? I came across a error in the FE code last week while debugging with Cara which was caused due to a missing optional chaining operator, as the code was trying to pass down the site?.XX variable before the site object was loaded in memory, causing the site to crash.

Reply:
Good question, Sarah and I came across the same issue last week. The button for the mobile menu, along with most of the Navbar, is now wrapped in a check for site loading so it shouldn't be an issue. I will double check it on Dev once it is merged before passing to QA. Thanks!